pages/moderation-search/moderation-search.scss

```diff
@@ -8,6 +8,28 @@
   height: 100%;
 
   .Select-control {
-    border-color: $input-border-color;
+    border-color: $black;
```

Comment:
Are these using the same rules as the ones we apply to homepage's select dropdown? If so we should think of a way for them to share the same rule set instead of having duplicate.

Reply:
Yeah, it looks like the dropdown is mainly sharing `height`,`border-radius`, and `border-color`. You think I should create a mixin for them?

Reply:
Ah gotcha. For this to not become a blocker - as a followup we should align styling of all dropdowns on Pulse.

Reply:
Ya, good idea.